### ISSUE TITLE
Zenith angle dependant albedo

### ISSUE THREAD

**Issue by Diftin** [Eawag-AppliedSystemAnalysis/Simstrat#33]
Add a zenith angle dependent albedo to Simstrat. Could for example be done analogous to GLM, where the following formula is implemented:
Albedo = 0.08 - 0.02 * sin(2*Pi * kDays/365 +- Pi/2);

For the northern hemisphere, the halfPi is subtracted and for the southern hemisphere halfPi is added.

**Comment by Diftin** [comment]
I implemented monthly changing water albedo using the data of Grishchenko (in Cogley 1979). For the Northern hemisphere I used the data in Table 6 and for the Southern hemisphere I shifted the data by 6 months. Currently, there is no interpolation implemented, meaning that the albedo changes abruptly from March 31 to April 1 for example.
In the Simstrat config file I now ask for the starting year of the time format used for forcing files (usually 1900 or 1981) in order to correctly assign calendar months for each simulation day.

Feedback is very welcome!
